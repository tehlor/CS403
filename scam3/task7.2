(define x 0)
(sdisplay 5 (mystery x)) (println)
(sdisplay 5 (ps-mystery x)) (println)
(sdisplay 5 (acc-mystery x)) (println)
(sdisplay 5 (super-mystery x)) (println)
(symbolic-mystery)
