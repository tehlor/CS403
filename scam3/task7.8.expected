(1.0000000000,-40.500000000,273.37500000,-738.11250000,1067.6270089,...)
(1.0000000000,-39.500000000,233.87500000,-504.23750000,563.38950893,...)
(-4.2258064516,34.385135135,-67.835218978,57.671452068,-32.075696051,...)
(1.0000000000,-4.2258064516,6.3599429928,-2.8310492056,-0.8770491617,...)
(mystery x) is $\cos x$
